app/models/user.rb

```diff
@@ -0,0 +1,4 @@
+class User < ActiveRecord::Base
+  validates_presence_of :username
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.